pkg/util/externalnode/externalnode.go

```diff
@@ -14,7 +14,15 @@
 
 package externalnode
 
-import "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+import (
+	"crypto/sha1" // #nosec G505: not used for security purposes
+	"encoding/hex"
+	"io"
+
+	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
+)
+
+const interfaceNameLength = 4
```

Comment:
Shall we refer to the suffix length in a Pod name used by Kubernetes?

Reply:
For the pods created by the same deployment, it seems to be 5. I have updated this part.